Finance/teller-track-spending.1h.sh

```diff
@@ -7,7 +7,7 @@
 # - teller-cli: https://github.com/sebinsua/teller-cli#from-release
 # - pcregrep: `brew install pcre`
 #
-# <bitbar.title>teller-track-spending</bitbar.title>
+# <bitbar.title>Track spending at UK banks</bitbar.title>
```

Comment:
Actually I just realised you don't have a description! Perhaps the title could be `Teller Spending Tracker` and the desc could be `Track your spending with the teller-cli and the Teller banking api`, or something similar?